pyproject.toml

```diff
@@ -68,10 +68,44 @@ write_to = "pytest_asyncio/_version.py"
 
 [tool.ruff]
 line-length = 88
+format.docstring-code-format = true
 lint.select = [
+  "B", # bugbear
+  "D", # pydocstyle
   "E", # pycodestyle
   "F", # pyflakes
-  "W", # pycodestyle
+  #  "FA100",  # add future annotations (not automated but would permit to update 'Union' via UP)
```

Comment:
There's probably a tool that would allow us to not have to copy paste ``from __future__ import annotations`` to the top of 30 files 😄

Reply:
There was indeed